pkgs/by-name/dc/dc3dd/package.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, fetchzip
+, perl538Packages
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dc3dd";
+  version = "7.3.1";
+
+  src = fetchzip {
+      url = "http://downloads.sf.net/sourceforge/dc3dd/dc3dd-${finalAttrs.version}.zip";
+      hash = "sha256-SYDoqGlsROHX1a0jJX11F+yp6CeFK+tZbYOOnScC6Ig=";
+  };
+
+  postPatch = ''
+    chmod +x configure
+  '';
+
+  buildInputs = [ perl538Packages.LocaleGettext ];
+
+  buildPhase = ''
+    runHook preBuild
+    if stdenv.hostPlatform.isi686 && stdenv.hostPlatform.isx86_64
+    then
+      ./configure --prefix=$out
+    else
+      ./configure --build=arm --prefix=$out
+    fi
+    make
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    make PREFIX=$out install
```

Comment:
Perhaps `makeFlags` could help here.

Reply:
Fixed. I tried to use `configureFlags = [ "--prefix=$out" ];` but during the compiling it didn't generate the files inside the `result` folder during my tests, so I chose to not use it.

Reply:
Usually `--prefix=$out` is automatic